src/integrationTest/resources/logback-test-base.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we need to override logback configuration for integration tests?

Reply:
ooh, embarrassing. i started typing output captures but my heart said it's not gonna work. so ended up over-engineering